app/views/design_methods/_form.html.haml

```diff
@@ -11,6 +11,26 @@
     border-bottom: 1px solid #ddd;
     padding: 20px;
   }
+  .tool-help {
```

Comment:
@thuangy @james--jiang Can you make these css classes more generic as you did with ".more-info-headers" rather than ".tool-help-container" etc. Since they apply to all the edit fields they shouldn't be named after the tools field.